include/crm_internal.h

```diff
@@ -69,6 +68,9 @@
 // @COMPAT Deprecated since 2.1.7
 #define PCMK__XE_DIFF_REMOVED           "diff-removed"
 
+// @COMPAT Deprecated since 2.1.8
+#define PCMK__XE_PARAM                  "param"
```

Comment:
Of course this wouldn't be deprecated

Reply:
Thanks (easy to forget to update things like this)